internal/website/content/howto/pubsub/publish.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Publish Messages to a Topic"
+date: 2019-03-26T09:44:15-07:00
+draft: true
+weight: 1
+---
+
+Publishing a message to a topic with the Go CDK takes two steps:
+
+1. Open the topic with the Pub/Sub provider of your choice (once per topic).
+2. Send messages on the topic.
+
+The second step is the same across all providers because the first step
+creates a value of the portable [`*pubsub.Topic`][] type. Publishing looks
+like this:
+
+```go
+err := topic.Send(ctx, &pubsub.Message{
+    Body: []byte("Hello, World!\n")
+    Metadata: map[string]string{
+        "Foo": "Bar",
```

Comment:
Consider using a realistic example of metadata. I sure would like to see one myself.

Reply:
Not sure if it's realistic enough, but Contribute Bot uses metadata to capture headers from the webhook (convenience more than anything else): https://github.com/google/go-cloud/blob/f4bf16298678fbb3b86c91e31a7f3bcaf4019d41/internal/contributebot/webhook/main.go#L93-L103

However, I'm not sure if that example is helpful unless one has the context of how GitHub webhooks work.

Reply:
Yeah, too specific. I was hoping you could think of something that would be more universal. Some quick ideas:
```
language: en
importance: high
```